tools/java/src/com/twitter/bazel/checkstyle/heron_header.txt

```diff
@@ -1,13 +1,13 @@
-// Copyright 2016 Twitter. All rights reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+\/\/ Copyright \d\d\d\d Twitter\. All rights reserved\.
```

Comment:
While you're at it, could we switch the regex check to allow 1-2 spaces between the leading `//` and the text? My IDE puts 2 spaces after `//` and it drives me nuts. I can't find a way to change it.